src/test/kotlin/com/exactpro/th2/codec/fix/orchestra/TestFixOrchestraCodec.kt

```
@@ -98,6 +98,28 @@ class TestFixOrchestraCodec {
         ))
     }
 
+    @Test
+    fun `decodes order cancel request message `() { //rule OrigClOrdIDnotrequiredwhenOrderIDexists [!exists OrderID]
```

Shouldn't we check the case when the rule should report an error instead?